lib/std/packedsets.nim

```diff
@@ -438,6 +438,9 @@ proc `=copy`*[A](dest: var PackedSet[A], src: PackedSet[A]) =
       dest.data[h] = n
       it = it.next
 
+proc `=dup`*[A](src: PackedSet[A]): PackedSet[A] {.inline.} =
+  `=copy`(result, src)
```

Comment:
Is there a reason this can't be the default implementation?

Reply:
It's upside down, `=dup` is more fundamental than `=copy` and `=copy` should be generated from `=dup`.

Reply:
> Is there a reason this can't be the default implementation?

It is feasible. I had it implemented in my PR, which pass tests without breaking old packages as well.